src/utils/crypto_utils/src/root_key_util.c

```diff
@@ -1,5 +1,5 @@
 /**
- * @file root_key_utility.c
+ * @file root_key_util.c
```

Comment:
Would be at some point to add a script to run as git precommit hook to catch differences between this and the actual file name.  I added [a task](https://microsoft.visualstudio.com/OS/_workitems/edit/46158785) for this